packages/core/src/RPA/core/windows/locators.py

```diff
@@ -43,9 +42,11 @@ class WindowsElement:
     """Represent Control as dataclass"""
```

Comment:
I feel this `WindowsElement` class doesn't fully work with `get_element` because the `Window` one doesn't share the same context and library construct (with `DynamicCore`) as this one. Inspector example:
1. Getting the windows elements [object](https://github.com/robocorp/inspector/blob/master/inspector/windows_driver.py#L75)
2. [Calling](https://github.com/robocorp/inspector/blob/master/inspector/windows_driver.py#L167-L168) `get_element`
3. If `"handle"` or `"executable"` are used as search params (strategy), then [listing windows](https://github.com/robocorp/rpaframework/blob/2dc3bf93b115f4c8816d48aa2b5b8b8a2c9c79c5/packages/core/src/RPA/core/windows/locators.py#L221) will fail as that method isn't found in the object instantiated at step **1.**

Solution: Following the pattern seen with the windows library [here](https://github.com/robocorp/rpaframework/blob/2dc3bf93b115f4c8816d48aa2b5b8b8a2c9c79c5/packages/windows/src/RPA/Windows/__init__.py#L28) by [registering](https://github.com/robocorp/rpaframework/blob/2dc3bf93b115f4c8816d48aa2b5b8b8a2c9c79c5/packages/windows/src/RPA/Windows/__init__.py#L488-L495) all libraries to be included.

Reply:
Post-huddle with Matti, we like this solution, but let's try using it without importing the entire **robotframework** code:
- Maybe **robotframework-pythonlibcore** is enough, but still depends on `robot` import
- Extracting the design pattern itself from the above

Reply:
Another solution would be to just move all the methods into a single class and use them from there in the inspector, but would pose design issues when inheriting those inside the windows library.

Reply:
The winning solution remains to not depend on **robotframework** package, while having the keyword methods still split (separate concerns) and then combining all of these in a single class by inheriting the core library `WindowsEelements` class from all of them.

Reply:
That's a pretty good solution 👍.